### PR TITLE
feature/242-draft_for_security_implementation

### PR DESCRIPTION
**Pull request by odzio33**
#### Checklist template for LEADERS

&#x2611; | &#x2610;

&#x2610; New code is covered with tests and coverage requirements are met.
&#x2610; Build is not failing.
&#x2610; All tests are passing.
&#x2610; All conflicts are resolved and the branch is rebased on top of the up-to-date parent branch.
&#x2610; All issue requirements are implemented.
&#x2610; At least two reviewers are assigned.
&#x2610; The Pull request is linked to the related issue.
&#x2610; Correct branch name. Convention [https://github.com/bycza-zagroda/track-expenses-app-documentation/blob/develop/githubwork/GITHUB_WORK.md]
&#x2610; Correct unit test name - suffix 'Test' in the end.
&#x2610; Correct integration test name - suffix 'IT' in the end.
&#x2610; Positive and negative test paths have been implemented.

### CODE REVIEW

**Review by dawciobiel — DISMISSED** [review]
Looks good to me

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/AuthController.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    @PostMapping("/register")
+    public ResponseEntity<Void> registerUser(@Valid @RequestBody AuthRegisterDTO authRegisterDTO) {
+        //TODO
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+    @PostMapping("/refresh")
+    public ResponseEntity<Void> refreshToken(@CookieValue(name = "refresh_token") String refreshToken) {
```
Comment (poneciak57) [inline src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/AuthController.java:43]: /refresh Endpoint should return Access token too.

**Review by poneciak57 — CHANGES_REQUESTED** [review]

File: src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/AuthController.java (hunk 1 above)
Comment (odzio33) [reply src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/AuthController.java]: done

**Review by dawciobiel — APPROVED** [review]
Looks good to me

**Review by poneciak57 — APPROVED** [review]
Looks good to me

**Pull request merged**